src/app/fdctl/config/default.toml

```diff
@@ -493,6 +493,9 @@ dynamic_port_range = "8000-10000"
         # should be unused.
         tx_buf_size = 4096
 
+        # How long does an idle connection remain active before being closed
+        idle_timeout_ms = 2500
```

Comment:
Minor but can we use `idle_timeout_millis`, ms can be mistaken for micros by a user.

Reply:
Also can you elaborate the comment for a validator operator (non-developer) like the other ones.

Reply:
It's not new - it was just unset. But I'm happy to rename it